### PR TITLE
feat (api, ui): Rename worfklow node

### PR DESCRIPTION
**Pull request by sguiheux**

### CODE REVIEW

File: ui/src/assets/i18n/fr.json [hunk 1]
```diff
@@ -566,6 +566,10 @@
   "workflow_node_trigger_condition_no": "Il n'y a aucune condition de déclenchement",
   "workflow_node_trigger_manual" : "Trigger manuel",
   "workflow_node_trigger_title" : "Trigger à partir du pipeline {{pip}}",
+  "workflow_node_name_warning" : "Warnings",
+  "workflow_node_name_warning_modal_title" : "Attention, le nom du noeud est utilisé",
+  "workflow_node_name_warning_modal_hooks" : "Le nom du noeud est utilisé dans les hooks suivant : ",
+  "workflow_node_name_warning_modal_triggers" : "Le nom do noeud est utilisé dans les triggers suivant : ",
```
Comment (yesnault) [inline ui/src/assets/i18n/fr.json]: Le nom du noeud

File: ui/src/app/model/workflow.model.ts [hunk 2]
```diff
@@ -209,9 +230,45 @@ export class WorkflowNode {
...
+                        }
+                    });
+                }
+            });
+        }
+        if (n.triggers) {
+            n.triggers.forEach(t => {
+                if (t.conditions) {
+                    t.conditions.forEach(c => {
+                        console.log(c);
```
Comment (bnjjj) [inline ui/src/app/model/workflow.model.ts]: to delete

File: ui/src/assets/i18n/fr.json (hunk 1 above)
Comment (bnjjj) [inline ui/src/assets/i18n/fr.json]: do -> du

File: engine/api/workflow.go [hunk 3]
```diff
@@ -163,31 +163,33 @@ func (api *API) putWorkflowHandler() Handler {
...
-			var errHooks error
-			for _, s := range srvs {
-				code, errBulk := services.DoJSONRequest(&s, http.MethodPost, "/task/bulk", hooks, nil)
-				errHooks = errBulk
-				if errBulk == nil {
-					log.Debug("putWorkflowHandler> %d hooks created for workflow %s/%s (HTTP status code %d)", len(hooks), wf.ProjectKey, wf.Name, code)
-					break
+
+			//Perform the request on one off the hooks service
+			if len(hooks) > 0 {
```
Comment (fsamin) [inline engine/api/workflow.go]: delete this conditions, it's already checked

**Pull request merged**